### ISSUE TITLE
Opening .PCX files

### ISSUE THREAD

**Issue by RicoLeNecro** [nomacs/nomacs#616]
Hello there.

Not sure if this is a bug (it could be since feature request #443 is closed), but it seems that nomacs (or at least, version 3.16.1709.0 for Windows) cannot open PCX files.

When trying to open such a file, the program briefly displays in the bottom left corner a message saying "Sorry, I could not load: _filename_.pcx"

I'd attach a sample file, but github isn't letting me do it, even if I zip it. :(

**Comment by scrubbbbs** [comment]
PCX is supported via kimageformats-plugins, please try a current build and feel free to reopen.

**Comment by RicoLeNecro** [comment]
Thanks!
Still having an issue with some .PCX files not opening with version 3.17.2297, though it's not all of them.
Should I report the issue at https://github.com/KDE/kimageformats/issues/ then ?

**Comment by scrubbbbs** [comment]
> Should I report the issue at https://github.com/KDE/kimageformats/issues/ then ?

Yes, that would be helpful. But you should try latest or newer kimageformats, there were specifically several commits for PCX in 2023-2024, the last one in November '24.
